pom.xml

```diff
@@ -2386,6 +2386,9 @@ under the License.
 								<exclude>org.apache.flink.types.DoubleValue</exclude>
 								<exclude>org.apache.flink.types.FloatValue</exclude>
 								<exclude>org.apache.flink.types.NormalizableKey</exclude>
+								<!-- FLINK-35566 promote API in flink-core -->
+								<exclude>org.apache.flink.api.common.typeutils.TypeSerializerSchemaCompatibility</exclude>
+								<exclude>org.apache.flink.api.common.typeutils.TypeSerializerSnapshot#readVersionedSnapshot(org.apache.flink.core.memory.DataInputView,java.lang.ClassLoader)</exclude>
```

Comment:
Without these two `exclude`, these two errors will appear when running the `mvn deploy`:

`There is at least one incompatibility: org.apache.flink.api.common.typeutils.TypeSerializerSchemaCompatibility.compatibleAfterMigration():CLASS_GENERIC_TEMPLATE_CHANGED,org.apache.flink.api.common.typeutils.TypeSerializerSchemaCompatibility.compatibleAsIs():CLASS_GENERIC_TEMPLATE_CHANGED,org.apache.flink.api.common.typeutils.TypeSerializerSchemaCompatibility.compatibleWithReconfiguredSerializer(org.apache.flink.api.common.typeutils.TypeSerializer):CLASS_GENERIC_TEMPLATE_CHANGED,org.apache.flink.api.common.typeutils.TypeSerializerSchemaCompatibility.incompatible():CLASS_GENERIC_TEMPLATE_CHANGED,org.apache.flink.api.common.typeutils.TypeSerializerSnapshot.readVersionedSnapshot(org.apache.flink.core.memory.DataInputView,java.lang.ClassLoader):CLASS_GENERIC_TEMPLATE_CHANGED`

`There is at least one incompatibility: org.apache.flink.api.common.typeutils.TypeSerializerSnapshot.readVersionedSnapshot(org.apache.flink.core.memory.DataInputView,java.lang.ClassLoader):CLASS_GENERIC_TEMPLATE_CHANGED`